lib/satellite/controller_extensions.rb

```diff
@@ -5,6 +7,8 @@ module ControllerExtensions
     included do
       helper_method :current_user
       helper_method :user_signed_in?
+
+      before_action :skip_satellite_authentication, if: :should_skip_satellite_authentication?
```

Comment:
Are we sure we want this "skip" ability on all actions in the satellite client? One option would be a single satellite route that handles this and then redirects to the proper final location (that can be stored on the client).